dashboard/app/views/layouts/application.html.haml

```diff
@@ -23,6 +23,8 @@
     -# data needed for dcdo.js. This tag is needed on every page because our translation code is
     -# making use of the dcdo feature.
     %script{data: {'dcdo': DCDO.frontend_config.to_json}}
+    -# auth key needed for amplitude analytics.
+    %script{data: {'amplitude-api-key': CDO.safe_amplitude_api_key}}
```

Comment:
It's worth noting that if the key can't be found, `<script data-amplitude-api-key=""></script>` will be on the page but if the key is `nil` this element won't be on the page at all. This seems fine though because we can handle the element not existing

Reply:
Yeah, the JS handles both cases in basically the same way, but I do prefer the empty string one because it makes it easier to look for the script tag in the page during development. That tripped me up a few times looking for an attribute that didn't exist.